scripts/bundleSizeConfig.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
-  MIN_SIZE: 569,
-  MAX_SIZE: 609,
+  MIN_SIZE: 564,
+  MAX_SIZE: 614,
```

Comment:
Please can you account for the 5K increase in max bundle size on this PR?

Reply:
yeah, The russian package was too large
```
✖ Issues with service bundles:
Bundle size for Russian is too large at 610 kB. Please update thresholds in './scripts/bundleSizeConfig.js'
/Users/brumpj01/Code/simorgh/scripts/bundleSize.js:87
  throw new Error();
  ^
```